.travis.yml

```diff
@@ -1,34 +1,12 @@
-sudo: false
-language: cpp
+sudo: required
 
-matrix:
-  include:
-    - os: linux
-      compiler: clang
-      addons: &clang38
-        apt:
-          sources: ['llvm-toolchain-precise-3.8', 'ubuntu-toolchain-r-test']
-          packages: ['clang-3.8', 'libclang-3.8-dev']
-      env: COMPILER='clang++-3.8'
+services:
+  - docker
 
 install:
-  - pip install --user clang django nose
-  - gem install ffi
-  - DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
-  - mkdir ${DEPS_DIR} && cd ${DEPS_DIR}
-  - CMAKE_URL="http://www.cmake.org/files/v3.3/cmake-3.3.2-Linux-x86_64.tar.gz"
-  - mkdir cmake && travis_retry wget --no-check-certificate --quiet -O - ${CMAKE_URL} | tar --strip-components=1 -xz -C cmake
-  - export PATH=${DEPS_DIR}/cmake/bin:${PATH}
-  - cd ${TRAVIS_BUILD_DIR}
+  - docker build -t ffig_cont .
+  - docker ps -a
 
-before_script:
-  - export CXX=${COMPILER}
-  - export LD_LIBRARY_PATH=$(dirname $(find /usr/lib/llvm-3.8 -name libclang.so))
-
-script: ${TRAVIS_BUILD_DIR}/run_tests.sh
-
-notifications:
-  slack:
-    rooms:
-      - c-api:ZTxxqvEcr25iaaB8XBS7luaA#general
 
+script:
+  - docker run ffig_cont /bin/bash -c "export PATH=$PATH:/home/c-api-user/.local/bin && cd ../ && ./run_tests.sh"
```

Comment:
@jbcoe this is a bit of a dirty hack. For some reason, pip install adds nosetests executable to home/c-api-user/.local/bin